x-pack/plugins/apm/server/lib/service_map/get_service_map_service_node_info.ts

```diff
@@ -71,32 +73,24 @@ export async function getServiceMapServiceNodeInfo({
   };
 }
 
-async function getErrorMetrics({ setup, minutes, filter }: TaskParameters) {
-  const { client, indices } = setup;
-
-  const response = await client.search({
-    index: indices['apm_oss.errorIndices'],
-    body: {
-      size: 0,
-      query: {
-        bool: {
-          filter: filter.concat({
-            term: {
-              [PROCESSOR_EVENT]: 'error',
-            },
-          }),
-        },
-      },
-      track_total_hits: true,
-    },
+async function getErrorMetrics({
```

Comment:
Similar to the other comment: can we drop the "metrics" prefix and just call this `getErrorRate` or similar?

Reply:
Fixed.